integration-cli/docker_cli_links_test.go

```diff
@@ -15,6 +15,8 @@ import (
 )
 
 func TestLinksEtcHostsRegularFile(t *testing.T) {
```

Comment:
I'm confused -- which part of this test is assuming that the tests are running on the same host as the daemon?